lms/djangoapps/courseware/access.py

```diff
@@ -370,11 +400,13 @@ def _can_load_course_overview(user, course_overview):
         The user doesn't have to be enrolled in the course in order to have load
         load access.
     """
-    return (
-        not course_overview.visible_to_staff_only
+    response = (
+        _visible_to_nonstaff_users(course_overview)
         and _can_access_descriptor_with_start_date(user, course_overview, course_overview.id)
-    ) or _has_staff_access_to_descriptor(user, course_overview, course_overview.id)
+    )
 
+    return (ACCESS_GRANTED if (response or _has_staff_access_to_descriptor(user, course_overview, course_overview.id))
```

Comment:
parenthesis formatting convention is to place the open and closing paren on their own lines.